src/resource/__tests__/resource.ts

```diff
@@ -5,7 +5,9 @@ import {
   UserResource,
   UrlArticleResource,
 } from '../../__tests__/common';
-import { Resource, normalize, SimpleResource } from '..';
+import { normalize } from '../normal';
+import Resource, { isResponseEmpty } from '../Resource';
```

Comment:
are you using isResponseEmpty anywhere in this file?

Reply:
Oh no, my bad!
I was going to add more tests, but I decided to keep this PR small for now and do more tests in future PRs.
Let me remove `isResponseEmpty` for now.